### ISSUE TITLE
useTabs workspace setting does not override useTabs user setting

### ISSUE THREAD

**Issue by monteiz** [prettier/prettier-vscode#1856]
### Summary

I wanted to set a `useTabs` value specifically for a Workspace. I wanted the Workspace to use spaces instead of tabs, while leaving the User settings using tabs.

The problem is that Prettier keeps on formatting the code using tabs.

### Steps To Reproduce:

1.  In User settings, set useTabs to `true`
1.  In Workspace settings, set useTabs to `false`
1.  Format your code

### Expected result

The code should be formatted using spaces.

### Actual result

The code is formatted using tabs.

**VS Code Version**:
Version: 1.54.3
Commit: 2b9aebd5354a3629c3aba0a5f5df49f43d6689f8
Date: 2021-03-15T11:57:12.728Z
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin x64 18.7.0

**Prettier Extension Version**:

**OS and version**:
macOS 10.14.6 (18G7016)

**Comment by monteiz** [comment]
@ntotten Are you referring to the **Prettier: Use Tabs** setting? That is:

<img width="981" alt="Screenshot 2021-03-17 at 20 20 49" src="https://user-images.githubusercontent.com/853871/111525883-59e83d80-875e-11eb-85e2-701c96ec437b.png">

If it is not used, shouldn't it be removed?

**Comment by ntotten** [comment]
I am not able to reproduce this issue. A lot of the time these issues are caused by various version and config settings that are difficult to duplicate. Please provide a sample Github repo that I can use to duplicate this issue. For more details see [this guide](https://github.com/prettier/prettier-vscode/wiki/Writing-a-Good-Issue#5-provide-a-sample-github-repo).

**Comment by ntotten** [comment]
> Are you referring to the Prettier: Use Tabs setting?

No, the Prettier setting is used. The other one is the built-in setting. Prettier doesn't use that.

**Comment by monteiz** [comment]
@ntotten thanks for looking into this. Let me rephrase it.

First of all, I am exclusively talking about the `Prettier: useTabs` setting (hereinafter only `useTabs`).

Secondly:

"Workspace settings override user settings." [docs](https://code.visualstudio.com/docs/getstarted/settings)

But instead, the Prettier extension does the following (confirmed by the Prettier logs):

If User settings: `"useTabs": true`
and Workspace settings: `"useTabs": true`
--> `"useTabs": true` **OK** ✅

If User settings: `"useTabs": false`
and Workspace settings: `"useTabs": true`
--> `"useTabs": true` **OK** ✅

If User settings: `"useTabs": false`
and Workspace settings: `"useTabs": false`
--> `"useTabs": false` **OK** ✅

**And here is where the bug is:**

If User settings: `"useTabs": true`
and Workspace settings: `"useTabs": false`
--> `"useTabs": true` **WRONG** ❌

The last case is wrong, because `useTabs` should be `false`.

**Comment by ntotten** [comment]
If this is the case, it's a VSCode bug. We simply read config VS Code resolves.

**Comment by monteiz** [comment]
> If this is the case, it's a VSCode bug. We simply read config VS Code resolves.
> Can you provide a GitHub repo to use to reproduce?

Why do you need a repo to reproduce the issue, if you say that it is a VSCode bug?
